broker/test/test_broker/mocks/index.js

```diff
@@ -44,8 +44,20 @@ function init() {
   return exports;
 }
 
-function verify() {
+function verify(ignoreList = []) {
   /* jshint expr:true */
+  if(nock.pendingMocks().length > 0) {
+    if(nock.pendingMocks().length == ignoreList.length) {
```

Comment:
For better readability store `nock.pendingMocks()` in a variable.

Reply:
Sure. Making the changes